lib/icons/jsx-svg.jsx

```diff
@@ -8,16 +8,22 @@ import JsxParser from 'react-jsx-parser';
  * @param {Object} props - Component props.
  * @param {string} props.svg - The SVG string to render.
  * @param {string} [props.className] - The class name to apply to the SVG. **Note**: Make sure the SVG doesn't include a class name already!
+ * @param {boolean} [props.decorative] - If `true`, `aria-hidden="true"` is added to the SVG.
+ * @param {string} [props.customProps] - Custom props to add to the SVG.
+ * @param {Object} [props.customPropBindings] - Dynamic bindings for custom props.
  *
  * @returns {JSX.Element} The JsxSvg component.
  *
  * @example
  * <JsxSvg svg="<svg ..." />
  *
+ * @example
+ * <JsxSvg customProps='className={demo}' customPropBindings={{demo: demoAttr ? 'lorem' : 'ipsum}} svg="<svg ..." />
+ *
  * @preserve
  */
 export const JsxSvg = (props) => {
-	const { svg, className } = props;
+	const { svg, className, decorative, customProps, customPropBindings } = props;
```

Comment:
Instead of `decorative` maybe use something more in line with existing ARIA roles and data attributes like `ariaHidden`. This way we can implement aria props which are easy to remember

Reply:
Sure!